controllers/argocd/keycloak.go

```diff
@@ -1515,3 +1514,18 @@ func (r *ReconcileArgoCD) reconcileKeycloak(cr *argoproj.ArgoCD) error {
 
 	return nil
 }
+
+func restrictedContainerSecurityContext() *corev1.SecurityContext {
```

Comment:
Why are we creating a function for security context only for keycloak resource?

Reply:
Actually, I wanted to create a common function for all deployments but realized that some have slightly different `securityContext` configurations. So to keep things simpler, didn't proceed with it. We can revisit this when we work on refactoring again.